src/main/resources/assets/skyblocker/lang/en_us.json

```diff
@@ -87,10 +87,12 @@
   "skyblocker.config.dungeons.goldorWaypoints.waypointType": "Task waypoint type",
 
   "skyblocker.config.dungeons.devices": "Device Solvers (F7/M7)",
-  "skyblocker.config.dungeons.devices.solveLightsOn": "Solve Lights On",
-  "skyblocker.config.dungeons.devices.solveLightsOn.@Tooltip": "Highlights the correct levers to click in red",
   "skyblocker.config.dungeons.devices.solveSimonSays": "Solve Simon Says",
   "skyblocker.config.dungeons.devices.solveSimonSays.@Tooltip": "Highlights the correct button to click in green, and highlights the next one in yellow.",
+  "skyblocker.config.dungeons.devices.solveLightsOn": "Solve Lights On",
+  "skyblocker.config.dungeons.devices.solveLightsOn.@Tooltip": "Highlights the correct levers to click in red.",
+  "skyblocker.config.dungeons.devices.solveArrowPath": "Solve Arrow Path",
+  "skyblocker.config.dungeons.devices.solveArrowPath.@Tooltip": "Displays the number of rotations remaining for each arrow.",
```

Comment:
These are technically supposed to be alphabetical, but it's not currently enforced. So this is fine but for future reference.